components/camel-jetty/src/test/java/org/apache/camel/component/jetty/TwoCamelContextWithJettyRouteTest.java

```diff
@@ -60,7 +61,7 @@ public void process(Exchange exchange) {
 
         Exception ex = assertThrows(Exception.class,
                 () -> template.requestBody("direct:b", "Moon", String.class));
-        assertTrue(ex.getCause() instanceof NoHttpResponseException, "Should get the ConnectException");
+        assertTrue(ex.getCause() instanceof IOException, "Should get the IOException");
```

Comment:
Not a major thing ... but maybe you can use the opportunity to simplify this assertion and use `assertInstanceOf` instead.